### PR TITLE
Herbrun updates, Wilderness check for blighted food/potions + seed pod teleport

### PR DESCRIPTION
**Pull request by yairtu**
- added wilderness check for blighted food and prayer potions
-  seedpod teleport + wilderness check in playeralarm
- added support for toggle location in herbrun
-  added harmony support
- added added info to plugin as well as migrating guide to ConfigInformation panel
- added farming cape support

### CODE REVIEW

File: .run/Runelite.run.xml
Comment (chsami) [inline .run/Runelite.run.xml:1]: You can't be changing the runelite.run.xml for everyone else. Please remove this change

File: hs_err_pid23568.log
Comment (chsami) [inline hs_err_pid23568.log:1]: This file can be left out of your PR aswell

**Review by chsami — CHANGES_REQUESTED** [review]

File: .run/Runelite.run.xml
Comment (yairtu) [reply .run/Runelite.run.xml:1]: Im not sure how it got added again, I will add it to gitignore

File: hs_err_pid23568.log
Comment (yairtu) [reply hs_err_pid23568.log:1]: Should I delete it ?

Comment (chsami) [reply hs_err_pid23568.log:1]: yes

**Review by chsami — APPROVED** [review]

**Review by chsami — APPROVED** [review]

**Pull request merged**